easy-ui-react/src/utilities/css.ts

```diff
@@ -33,7 +34,23 @@ export function getComponentToken(
   };
 }
 
-export function getResponsiveToken(
+export function getComponentThemeToken(
+  componentName: string,
+  componentProp: string,
+  tokenSubgroup: string,
+  token?: string,
+) {
+  if (!token) {
+    return {};
```

Comment:
i know this bit isn't necessarily the focus of whats going on here, but out of curiosity, whats the thinking behind making token optional?

Reply:
i copied the original function from Polaris which did it this way, but i think the idea is that it allows for optional props in a component API can just be fed directly into this function without having to guard against it in the component itself